src/projects/service.js

```diff
@@ -1,7 +1,12 @@
 import Vue from 'vue';
 
 const ProjectsService = {
-  list: params => Vue.http.get(`${Vue.config.projectsUrlBase}/api/v1/en/projects`, { params }),
+  list: (language, params) => {
+    // Note : this is a patch due to /projects non-consistent language
+    // eslint-disable-next-line no-unused-vars
```

Comment:
This doesn't seem necessary..?

Reply:
Well, the linter was complaining about lang not being used :/

Reply:
But, like... It is used. Also, conflicts! 😄

Reply:
Will have another look at it while solving the conflict, but pretty sure I had the same reaction than you did